File: solutions/python/armstrong-numbers/1/armstrong_numbers.py
def is_armstrong_number(number):
    if number == 0:
        return True
    digits = len(str(number))
    num = number
    is_armstrong_number = 0
    for i in range(digits):
        is_armstrong_number += pow(num % 10, digits)
        num //= 10
    return is_armstrong_number == number

File: solutions/cpp/armstrong-numbers/1/armstrong_numbers.cpp
#include "armstrong_numbers.h"
#include<bits/stdc++.h>
using namespace std;
namespace armstrong_numbers {

// TODO: add your solution here
    bool is_armstrong_number(int num){
        int count{0};
        int temp = num;
        int check{0};
        for(; temp > 0; temp /= 10)count++;
        temp = num;
        for(; temp > 0; temp /= 10) check += pow(temp % 10, count);
        return check == num;
    }

}  // namespace armstrong_numbers
